tribe-ext-settings-import-export.php

```diff
@@ -0,0 +1,370 @@
+	                }
+                    elseif ( $_GET['action'] == 'reset_success' ) {
+	                    $msg = __( 'Reset successful.', 'PLUGIN_TEXT_DOMAIN' );
+	                    $notice_class = 'notice-success ';
+                    }
+                    elseif ( $_GET['action'] == 'reset_failed' ) {
+                        $msg = __( 'Settings reset failed.', 'PLUGIN_TEXT_DOMAIN' );
+                        $notice_class = 'notice-error ';
+                        }
+                    elseif ( $_GET['action'] == 'reset_no' ) {
+	                    $msg = __( 'Reset failed. Please enter "reset" in the text field to reset the settings.', 'PLUGIN_TEXT_DOMAIN' );
+	                    $notice_class = 'notice-error ';
+                    }
+
+	                echo '<div class="notice ' . $notice_class . ' is-dismissible"><p>' . $msg . '</p></div>';
+                } ?>
+
+				<div class="metabox-holder">
+					<div class="postbox">
+						<h3><span><?php _e( 'Export Settings', 'PLUGIN_TEXT_DOMAIN' ); ?></span></h3>
+						<div class="inside">
+							<p><?php _e( 'Export the setting of The Events Calendar, Event Tickets and add-ons for this site as a .json file. This allows you to easily import the configuration into another site.', 'PLUGIN_TEXT_DOMAIN' ); ?></p>
+							<form method="post">
+								<p><input type="hidden" name="tribe_sie_action" value="export_settings" /></p>
+								<p>
+									<?php wp_nonce_field( 'tribe_sie_export_nonce', 'tribe_sie_export_nonce' ); ?>
+									<?php submit_button( __( 'Export', 'PLUGIN_TEXT_DOMAIN' ), 'secondary', 'submit', false ); ?>
+								</p>
+							</form>
+						</div><!-- .inside -->
+					</div><!-- .postbox -->
+
+					<div class="postbox">
+						<h3><span><?php _e( 'Import Settings', 'PLUGIN_TEXT_DOMAIN' ); ?></span></h3>
+						<div class="inside">
+							<p><?php _e( 'Import the plugin settings from a .json file. This file can be obtained by exporting the settings on another site using the form above.', 'PLUGIN_TEXT_DOMAIN' ); ?></p>
+							<form method="post" enctype="multipart/form-data">
+								<p>
+									<input type="file" name="import_file"/>
+								</p>
+								<p>
+									<input type="hidden" name="tribe_sie_action" value="import_settings" />
+									<?php wp_nonce_field( 'tribe_sie_import_nonce', 'tribe_sie_import_nonce' ); ?>
+									<?php submit_button( __( 'Import', 'PLUGIN_TEXT_DOMAIN' ), 'secondary', 'submit', false ); ?>
+								</p>
+							</form>
+						</div><!-- .inside -->
+					</div><!-- .postbox -->
+
+                    <div class="postbox">
+                        <h3><span><?php _e( 'Delete / Reset Settings', 'PLUGIN_TEXT_DOMAIN' ); ?></span></h3>
+                        <div class="inside">
+                            <p><?php _e( 'Reset the plugin settings.', 'PLUGIN_TEXT_DOMAIN'); ?></p>
+                            <p style="font-weight: bold";><?php _e( 'Please note the following:', '' )?></p>
```

Comment:
We're probably trying to avoid enqueuing a stylesheet etc with these inline styles which is fine, but perhaps using `<strong>` would be cleaner and better in a case like this. If not, we should fix the trailing semi-colon:

```suggestion
                            <p style="font-weight: bold"><?php _e( 'Please note the following:', '' )?></p>
```

Reply:
...You are missing the `PLUGIN_TEXT_DOMAIN` placeholder on this line, which is fine but just don't remember to replace it once you do settle on a text domain!